xlive/H2MOD/Modules/PlayerRepresentation/PlayerRepresentation.cpp

```diff
@@ -112,7 +113,12 @@ namespace PlayerRepresentation
 		else
 		{
 			if (H2Config_spooky_boy && !Memory::IsDedicatedServer())
-				*Memory::GetAddress<e_character_type*>(0x51A67C) = character_type_skeleton;
+			{
+				for (byte user_index = 0; user_index <= k_number_of_users; ++user_index)
```

Comment:
out of bounds...

Reply:
also doesn't follow the existing coding style

`for (int <name>; <name> < <condition>; <name>++)`

Reply:
Forgot to change the <= to a < when swapping from the controller index enum to user indexes in these loops. 
Fixed